### PR TITLE
Fix (docker): Use `127.0.0.1` instead of `localhost` to test snappymail in `entrypoint.sh`

### PR DESCRIPTION
**Pull request by leojonathanoh**

### CODE REVIEW

**Comment by Orhideous** [comment]
Then it worth to drop `/etc/nginx/http.d/default.conf` as well. Or remove `listen [::]:80 default_server;` line.

File: .docker/release/files/etc/nginx/nginx.conf [hunk 1]
```diff
@@ -52,7 +52,6 @@ http {
 
   server {
     listen 8888;
-    listen [::]:8888;
```
Comment (leojonathanoh) [inline .docker/release/files/etc/nginx/nginx.conf:55]: Yep, did so here

Comment (leojonathanoh) [reply .docker/release/files/etc/nginx/nginx.conf:55]: If the tests pass it should be good.

Comment (Orhideous) [reply .docker/release/files/etc/nginx/nginx.conf:55]: OK, got it. Everything in `http.d` is not included.

Comment (leojonathanoh) [reply .docker/release/files/etc/nginx/nginx.conf:55]: yes 😄

**Comment by leojonathanoh** [comment]
something in master causing this to fail

**Comment by leojonathanoh** [comment]
```
 Warning: require_once(/snappymail/snappymail/v/2.29.4app/libraries/snappymail/integrity.php): Failed to open stream: No such file or directory in /snappymail/snappymail/v/2.29.4/setup.php on line 3

Fatal error: Uncaught Error: Failed opening required '/snappymail/snappymail/v/2.29.4app/libraries/snappymail/integrity.php' (include_path='/snappymail/snappymail/v/2.29.4/app/libraries/:.:/usr/local/lib/php') in /snappymail/snappymail/v/2.29.4/setup.php:3
Stack trace:
#0 /snappymail/snappymail/v/2.29.4/include.php(110): include()
#1 /snappymail/index.php(11): include('/snappymail/sna...')
#2 {main}
  thrown in /snappymail/snappymail/v/2.29.4/setup.php on line 3
+ exit 255
```

@the-djmaze seems like recent commits 288eedf6a1ec1c200b1d61c396d9fa3ce302e162 and 3bff3555a0373f6b82ab06db8d3c15c32b771d73 on master is causing `php /snappymail/index.php` to fail with these errors.

**Comment by the-djmaze** [comment]
I notice that you changed the docker file to run on PR
afbeelding

**Comment by the-djmaze** [comment]
https://github.com/the-djmaze/snappymail/commit/7dc4002fb47a7c72a6513b5eac1efabaf7e68af3

**Comment by leojonathanoh** [comment]
> I notice that you changed the docker file to run on PR

Yes, this is so we can tell if docker images work on that PR change. It's a workflow common pattern.

**Pull request merged**